lte/gateway/python/magma/mobilityd/BUILD.bazel

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+load("@subpar//:subpar.bzl", "par_binary")
+load("@python_deps//:requirements.bzl", "requirement")
+
+DEBS = [
```

Comment:
Question: Was this intended to be DEBS and not DEPS?

If we are able to deprecate the `par_binary` maybe this collapses back into a single target and we don't define this shared construct? But I like that you reduced duplication in this present PR by the use of this define. Nice job being DRY.

Reply:
> Question: Was this intended to be DEBS and not DEPS?

Yes, it's a typo.

> As mention before, we also think that Subpar is deprecated and could be removed or replaced by an alternative library